src/site/content/en/handbook/content-types/index.md

```diff
@@ -33,14 +33,14 @@ description: |
 ## Codelabs
 **Format:** Multi-step instructions paired with an embedded [sample app](/handbook/markup-sample-app)
 
-**Typical use case:** showing how to do small, fairly discrete development tasks
+**Typical use case:** Showing how to do small, fairly distinct development tasks
 
-**Placement on the site:** always associated with a post, which serves as the landing page for the codelab
+**Placement on the site:** Always associated with a post, which serves as the landing page for the codelab
 
-**Length:** typically less than 1,000–1,500 words
+**Length:** Typically less than 1,000–1,500 words
```

Comment:
the word limit seems arbitrary. Do you want to explain how you came up with this?

Reply:
@robdodson, how _did_ you and Meggin come up with the word limit? 😃

Reply:
Added a note to fix in the Google Doc, pending larger revision to Content Types post.